resource/js/vocab-search.js

```diff
@@ -96,9 +103,15 @@ const vocabSearch = Vue.createApp({
           result.hitType = 'alt'
           result.hit = this.renderMatchingPart(renderedSearchTerm, result.altLabel)
           result.hitPref = this.renderMatchingPart(renderedSearchTerm, result.prefLabel)
-        } else if ('prefLabel' in result) {
-          result.hitType = 'pref'
-          result.hit = this.renderMatchingPart(renderedSearchTerm, result.prefLabel)
+        } else {
+          if (this.notationMatches(renderedSearchTerm, result.notation)) {
+            const notationMatch = this.renderMatchingPart(renderedSearchTerm, result.notation)
+            result.hitType = 'notation'
+            result.hit = notationMatch
```

Comment:
```suggestion
            result.hit = this.renderMatchingPart(renderedSearchTerm, result.notation)
```
Hit could be set like with other result types

Reply:
Ah, that was a left over that I forgot to clean up, from having a buggy way of checking whether the hit was on notation or on preflabel.